app/renderer/actions/Inspector.js

```diff
@@ -191,14 +194,22 @@ export function applyClientMethod (params) {
       dispatch({type: METHOD_CALL_DONE});
 
       if (source && screenshot) {
+        let newSource = source;
+        if (source.includes('<html')) {
+          // Remove head and scripts which cause extra noise in the tree
+          newSource = source.replace(/<head.*?\/head>/s, '').replace(/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/g, '');
```

Comment:
Could extract this logic and left a unit test as an example?

Reply:
Removed this code and put it into `webviewHelpers.js` and used _proper_ DOM element removal